operators/community-windows-machine-config-operator/6.0.0/manifests/community-windows-machine-config-operator.6.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,344 @@
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: community-windows-machine-config-operator
+                    image: quay.io/openshift-windows/community-windows-machine-config-operator:community-4.11-6c3601f
+                    imagePullPolicy: IfNotPresent
+                    name: manager
+                    resources: {}
+                hostNetwork: true
+                nodeSelector:
+                  node-role.kubernetes.io/master: ""
+                priorityClassName: system-cluster-critical
+                serviceAccountName: windows-machine-config-operator
+                terminationGracePeriodSeconds: 10
+                tolerations:
+                  - effect: NoSchedule
+                    key: node-role.kubernetes.io/master
+                    operator: Exists
+                  - effect: NoExecute
+                    key: node.kubernetes.io/unreachable
+                    operator: Exists
+                    tolerationSeconds: 120
+                  - effect: NoExecute
+                    key: node.kubernetes.io/not-ready
+                    operator: Exists
+                    tolerationSeconds: 120
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - configmaps
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+          serviceAccountName: windows-machine-config-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: false
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - windows
+  maintainers:
+    - email: team-winc@redhat.com
+      name: Red Hat, Windows Container Support for OpenShift
+  maturity: preview
+  minKubeVersion: 1.24.0
+  provider:
+    name: Red Hat
+  replaces: community-windows-machine-config-operator.v5.1.2
```

Comment:
I don't see a replaces field set in the csv produced in the latest artifacts, however I see it here. 
I don't think we need it for the a major version any reason to add it manually?